workflow/envs/flair.yml

```
@@ -2,4 +2,4 @@ channels:
     - conda-forge
     - bioconda
 dependencies:
-    - flair>=2.0.0
\ No newline at end of file
+    - flair=2.0.0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)